test/integration/src/test/java/io/pravega/test/integration/endtoendtest/EndToEndReaderGroupTest.java

```diff
@@ -444,6 +448,171 @@ public void testGenerateStreamCutsWithScaling() throws Exception {
         assertEquals(expectedSegments, scMap.get(stream).asImpl().getPositions().keySet());
     }
 
+    @Test
+    public void testCreateSubscriberReaderGroup() throws InterruptedException, ExecutionException {
+        StreamConfiguration config = getStreamConfig();
+        LocalController controller = (LocalController) controllerWrapper.getController();
```

Comment:
Cleanup

Reply:
Please fix this everywhere else in your code and for all objects that have a `shutdown` or `close` method.